rotkehlchen/premium/premium.py

```diff
@@ -822,6 +834,13 @@ def fetch_limits(self) -> UserLimits:
         log.debug(f'Fetched user limits from server: {self._cached_limits}')
         return self._cached_limits
 
+    def get_capabilities(self) -> dict[str, bool]:
+        limits = self.fetch_limits()
+        return {
+            feature_label: limits.get(feature_label, False)  # type: ignore # default to False in case we deprecate the key. Ignore type because mypy sees builtins.object instead of bool https://github.com/python/mypy/issues/19961
```

Comment:
Wouldnt' setting the `PREMIUM_CAPABILITIES_KEYS` explicitly work as the issue says ?